client/one-time-payment/react/src/context/sdkContext.tsx

```diff
@@ -35,13 +36,13 @@ export const PayPalSDKProvider: React.FC<PayPalSDKProviderProps> = ({
 
   const initSdkInstance = async () => {
     const sdkInstance: SdkInstance = await window.paypal.createInstance({
-      clientToken,
+      clientToken: clientToken!,
```

Comment:
Why do we need this type assertion? I guess more broadly why is `clientToken` optional in `PayPalSDKProviderProps` again?

Reply:
Wondering the same, and if it needs to be optional would it be better to wrap this block in a conditional statement, rather than allowing it to error?

Reply:
I agree that seeing `clientToken!` feels wrong. What do you think about this new approach in this commit? https://github.com/paypal-examples/v6-web-sdk-sample-integration/pull/33/commits/8f6f0bec0a754cb1deeb2b64b4ba73ae7219d4ac

Best I can tell, we have clientToken as optional for the `<PayPalSDKProvider>` component so you can async fetch the clientToken in the same parent component and still show child content in the provider while clientToken is fetching:

https://github.com/paypal-examples/v6-web-sdk-sample-integration/blob/main/client/one-time-payment/react/src/App.tsx#L27-L50